### PR TITLE
Add model card for singbert.

### PR DESCRIPTION
**Pull request by zyuanlim**
Adding a model card for singbert- bert for singlish and manglish.

### CODE REVIEW

**Review by JetRunner — APPROVED** [review]
This very nice lah! Thanks for ur contribution :)

File: model_cards/zanelim/singbert/README.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
+language: english, chinese, malay
```
Comment (JetRunner) [inline model_cards/zanelim/singbert/README.md:2]:
```suggestion
language: en
```
We now use ISO codes. Also, I don't personally think Chinese/Malay should be added here (cuz Singlish is not often considered a Chinese/Malay dialect)

Comment (zyuanlim) [reply model_cards/zanelim/singbert/README.md]: Thanks for your suggestion! Yes i think it makes sense :))

**Comment by zyuanlim** [comment]
@JetRunner Just a final addition of couple of more examples and customized the widget inputs, good to go :)

**Comment by zyuanlim** [comment]
Not sure if I should bring this up here or raise a new issue, but when i tested my widget (hosted inference), i got this error: 
```
Can't load config for 'zanelim/singbert'. Make sure that: - 'zanelim/singbert' is a correct model identifier listed on 'https://huggingface.co/models' - or 'zanelim/singbert' is the correct path to a directory containing a config.json file
```
However the config file is there as confirmed by
```
>>> transformers-cli s3 ls
singbert/config.json             2020-08-24T05:54:28.000Z "6004cb287370530537f4076c9cf7fdbe"       471 
singbert/pytorch_model.bin       2020-08-24T05:53:37.000Z "c060a644d84e55b2f93aa67fb1f35956" 440509997 
singbert/special_tokens_map.json 2020-08-24T05:54:23.000Z "8b3fb1023167bb4ab9d70708eb05f6ec"       112 
singbert/tf_model.h5             2020-08-24T05:52:23.000Z "45a8eea544f73079768bb136fe3d0a27" 536061440 
singbert/tokenizer_config.json   2020-08-24T05:54:25.000Z "8b3fb1023167bb4ab9d70708eb05f6ec"       112 
singbert/vocab.txt               2020-08-24T05:53:32.000Z "767659dd848f37f6937a0ffb833ee6b1"    224170 
```

**Comment by JetRunner** [comment]
> Not sure if I should bring this up here or raise a new issue, but when i tested my widget (hosted inference), i got this error: 
> 
> ```
> 
> Can't load config for 'zanelim/singbert'. Make sure that: - 'zanelim/singbert' is a correct model identifier listed on 'https://huggingface.co/models' - or 'zanelim/singbert' is the correct path to a directory containing a config.json file
> 
> ```
> 
> However the config file is there as confirmed by
> 
> ```
> 
> >>> transformers-cli s3 ls
> 
> singbert/config.json             2020-08-24T05:54:28.000Z "6004cb287370530537f4076c9cf7fdbe"       471 
> 
> singbert/pytorch_model.bin       2020-08-24T05:53:37.000Z "c060a644d84e55b2f93aa67fb1f35956" 440509997 
> 
> singbert/special_tokens_map.json 2020-08-24T05:54:23.000Z "8b3fb1023167bb4ab9d70708eb05f6ec"       112 
> 
> singbert/tf_model.h5             2020-08-24T05:52:23.000Z "45a8eea544f73079768bb136fe3d0a27" 536061440 
> 
> singbert/tokenizer_config.json   2020-08-24T05:54:25.000Z "8b3fb1023167bb4ab9d70708eb05f6ec"       112 
> 
> singbert/vocab.txt               2020-08-24T05:53:32.000Z "767659dd848f37f6937a0ffb833ee6b1"    224170 
> 
> ```

Don't worry. It's sometimes flaky. As long as you can load the model with from_pretrained and you are good to go. 😉

**Pull request merged**